src/Features/CSharp/Portable/ConvertSwitchStatementToExpression/ConvertSwitchStatementToExpressionDiagnosticAnalyzer.Analyzer.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Microsoft.CodeAnalysis.CSharp.ConvertSwitchStatementToExpression
+{
+    internal sealed partial class ConvertSwitchStatementToExpressionDiagnosticAnalyzer
+    {
+        private sealed class Analyzer : OperationVisitor<object, AnalysisResult>
+        {
+            private Analyzer()
+            {
+            }
+
+            public static AnalysisResult Analyze(ISwitchOperation operation)
+            {
+                return new Analyzer().VisitSwitch(operation, argument: null);
+            }
+
+            public override AnalysisResult VisitSwitch(ISwitchOperation operation, object argument)
```

Comment:
doc what 'argument' is for.

Reply:
comes from the base. can't do anything about it.

Reply:
can you call it "unused"?

Reply:
only on overrides, for instance line 35 can't be changes. so I think it's better to leave it as-is.

Reply:
since hte majority *are* overrides, i'd be ok with that :)  just document yours as unused, and in the call to 'Visit' just say "pass null for 'argument' since we're not passing along any data during computation" (or something to that effect).

I don't feel hugely strongly about this.  But it def confused me as it felt like tehre was some relation here between the "argument"s and how this feature worked.